practice-two/src/script/modules/addNote.js

```diff
@@ -11,22 +13,20 @@ const discardButton = document.querySelector('.modal-footer .btn-danger');
 const saveButton = document.getElementById('btn-n-save');
 const noteForm = document.getElementById('addnotesmodalTitle');
 
-const titleError = document.getElementById('title-error');
-const descriptionError = document.getElementById('description-error');
-
 const categoryBusinessCheckbox = document.getElementById('category-business');
 const categorySocialCheckbox = document.getElementById('category-social');
 const categoryTravelCheckbox = document.getElementById('category-travel');
 
-categoryBusinessCheckbox.addEventListener('click', checkbox);
-categorySocialCheckbox.addEventListener('click', checkbox);
-categoryTravelCheckbox.addEventListener('click', checkbox);
+categoryBusinessCheckbox.addEventListener('click', setNoteCategory);
+categorySocialCheckbox.addEventListener('click', setNoteCategory);
+categoryTravelCheckbox.addEventListener('click', setNoteCategory);
 
 async function addNoteAndRender() {
-  const title = noteTitleInput.value.trim();
-  const description = noteDescriptionInput.value.trim();
+  const formData = new FormData(noteForm);
+  const title = formData.get('title').trim();
+  const description = formData.get('description').trim();
 
-  if (title && description && selectedCategory) {
+  if (isFormValid(formData)) {
```

Comment:
Don't you apply from validation schema in this practice?

Reply:
I will apply it to the next practice